frontend/src/components/organization/organization-card.js

```diff
@@ -104,7 +108,7 @@ const OrganizationCard = ({ className, organization, ...rest }) => {
           >
             { organization.Projects && organization.Projects.map(p =>
               (<Chip key={ p.id } style={ { marginLeft: 10, marginBottom: 10 } } size='medium' clickable onClick={ () => {
-                window.location.href = '/#/organizations/' + organization.id + '/projects/' + p.id
+                window.location.href = `/#/organizations/${organization.id}/projects/${p.id}/${slugify(p.name)}`
```

Comment:
@VictorAlessander here it should have the organization name as part of the slug as in the specification of the task:
`https://gitpay.me/#/organizations/1/worknenjoy/projects/4/gitpay`

Reply:
Adjusted.